src/main/java/frc/robot/trajectory/AutoPaths.java

```diff
@@ -84,13 +86,115 @@
 	 */
 	public static ArrayList<Translation2d> r3HubPath() {
 		ArrayList<Translation2d> path = new ArrayList<>();
-		// path.add(new Translation2d(249.300, 85.490));
-		path.add(new Translation2d(278.800, 114.990));
-		// path.add(new Translation2d(249.300, 85.490));
+		path.add(new Translation2d(275.188, 111.126));
 		path.add(new Translation2d(227.904, 65.860));
 		path.add(new Translation2d(149.215, 8.764));
-		// path.add(new Translation2d(191.230, 0));
-		path.add(new Translation2d(62, -22));
+		path.add(new Translation2d(60, -27));
+		return path;
+	}
+
+	/**
+	 * The blue 1 ball path.
+	 * @return an arraylist containing the key points
+	 */
+	public static ArrayList<Translation2d> b1BallPath() {
+		ArrayList<Translation2d> path = new ArrayList<>();
+		path.add(Constants.PP_B1_START_POINT);
+		path.add(new Translation2d(-62, 22));
+		path.add(new Translation2d(-150, -3));
+		path.add(new Translation2d(-225, -3));
+		return path;
+	}
+
+	/**
+	 * The blue 1 "hub" path.
+	 * @return an arraylist containing the key points
+	 */
+	public static ArrayList<Translation2d> b1HubPath() {
+		ArrayList<Translation2d> path = new ArrayList<>();
+		path.add(new Translation2d(-225, -3));
+		path.add(new Translation2d(-225, -3));
+		return path;
+	}
+
+	/**
+	 * The blue 2 ball path.
+	 * @return an arraylist containing the key points
+	 */
+
+	public static ArrayList<Translation2d> b2BallPath() {
```

Comment:
For Monterey: We should just have global public static ArrayList instances for these paths, not creating a new instance every time we look up a path.